src/main/java/lotto/model/Lotto.java

```diff
@@ -0,0 +1,54 @@
+package lotto.model;
+
+import java.util.List;
+import lotto.constant.ErrorMessage;
+import lotto.validator.LottoValidator;
+
+public class Lotto {
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        if (!lengthValidate(numbers) || !rangeValidate(numbers) || !duplicationValidate(numbers)) {
+            throw new IllegalArgumentException();
+        }
```

Comment:
여기서는 함수에서 분리할 필요 없이 `Validate` 하기만 하면 좋을 것 같아요
```suggestion
        LottoValidator.containsSixNumbers(numbers.size());
        LottoValidator.haveValidateRange(numbers);
        LottoValidator.haveDuplicateNumber(numbers);
```